crates/sqllib/src/lib.rs

```diff
@@ -46,6 +46,7 @@ pub use variant::*;
 
 pub use array::Array;
 pub use binary::ByteArray;
+pub use dec::SqlDecimal;
```

Comment:
There's already `pub use dec::*` above. This must be redundant.

Reply:
why doesn't clippy complain?